### PR TITLE
Add RoR reference projects to backend/readme.md #411

### LINKED ISSUE #411: Add RoR reference projects to backend/readme.md
To help with any questions, these projects have been developed in ruby on rails and they will be used as references:

• Chatwoot: https://www.chatwoot.com/
• Repository: https://github.com/chatwoot/chatwoot

• Dev.to: https://dev.to/
• Repository: https://github.com/forem/forem

• Gitlab: https://about.gitlab.com/
• Repository: https://github.com/gitlabhq/gitlabhq

Note: If you find some project that can be used as reference too, you can add.

### PR DESCRIPTION
**Pull request by philipeleandro**
closes #411

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,10 +72,24 @@ Please check [CONTRIBUTING.md](https://github.com/espoo-dev/espoo-dev/blob/main/
   - Login, Logout, Register
   - List surveys
 
+## Reference projects
+
+To help with any questions, these projects have been developed in ruby on rails and they will be used as references:
```
Comment (edimossilva) [inline README.md:77]: You can remove this line. It is good to keep readme files small

**Review by edimossilva — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -79,3 +79,4 @@ Please check [CONTRIBUTING.md](https://github.com/espoo-dev/espoo-dev/blob/main/
 - Answers reports
 - Public surveys
 - Surveys protected by password
+
```
Comment (edimossilva) [inline README.md:82]: @philipeleandro remove this extra empty line

**Review by edimossilva — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (philipeleandro) [reply README.md]: @edimossilva It was removed.

File: README.md [hunk 3]
```diff
@@ -78,4 +78,4 @@ Please check [CONTRIBUTING.md](https://github.com/espoo-dev/espoo-dev/blob/main/
 - Free text, image, audio, video answers
 - Answers reports
 - Public surveys
-- Surveys protected by password
+- Surveys protected by password
```
Comment (edimossilva) [inline README.md:81]: @philipeleandro please remove this change

**Review by edimossilva — COMMENTED** [review]
There is one small change on readme.md that shouldn't be there

**Review by edimossilva — APPROVED** [review]
Nice work!

**Pull request merged**